templates/financial-services/resources.html

```diff
@@ -27,239 +31,309 @@ <h2>Canonical &ndash; Trusted source of open source for financial services</h2>
       </div>
-      <div class="col-7">
+      <div class="col-5">
         <p>Private cloud technology allows financial institutions to drive business agility and to rapidly respond to changing needs dictated by users, regulations, and the market. A private cloud is an integral part of a hybrid multi-cloud strategy for financial services organisations. It enables financial institutions to derive competitive advantage from agile implementations without incurring the security and business risks of a public cloud.</p>
       </div>
+      <div class="col-3 u-hide--small u-hide--medium">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/7debd05e-WH4.png",
+          alt="",
+          width="1654",
+          height="2339",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
     </div>
   </article>
-
   <hr class="u-fixed-width is-muted">
 
   <article class="p-strip is-shallow">
     <div class="row">
-      <div class="col-5">
+      <div class="col-4">
         <h3 class="p-heading--4"><a href="/engage/finserv-multi-cloud">Multi-cloud fundamental to financial services transformation</a></h3>
       </div>
-      <div class="col-7">
-        <p>Faced with a market transforming at the hands of new, cloud-native competitors, financial services businesses are actively engaging in their own transformation. They are adopting and integrating multiple cloud platforms and seeking to derive business value directly from emerging technologies such as artificial intelligence (AI), blockchain and container-based application architecture.</p>
-      </div>
-    </div>
-  </article>
-  
-  <hr class="u-fixed-width is-muted">
-
-  <article class="p-strip is-shallow">
-    <div class="row">
       <div class="col-5">
-        <h3 class="p-heading--4"><a href="/engage/ubuntu-finance-sector-whitepaper">9 out of top 10 US &amp; European financial institutions use Ubuntu</a></h3>
-        <p class="p-heading--5">Find out why multi-cloud is fundamental to financial services transformation</p>
+        <p>Faced with a market transforming at the hands of new, cloud-native competitors, financial services businesses are actively engaging in their own transformation. They are adopting and integrating multiple cloud platforms and seeking to derive business value directly from emerging technologies such as artificial intelligence (AI), blockchain and container-based application architecture.</p>
       </div>
-      <div class="col-7">
-        <p>Financial institutions are increasingly pressed for agility and velocity to adapt to changing market conditions, increased customer expectations while satisfying regulatory and compliance requirements.</p>
-        <p>Canonical helps financial institutions drive infrastructure efficiency and operational agility with multi-cloud, hybrid cloud and bare-metal solutions.</p>
+      <div class="col-3 u-hide--small u-hide--medium">
```

Comment:
[9 out of top 10 US & European financial institutions use Ubuntu](https://ubuntu.com/engage/ubuntu-finance-sector-whitepaper) Missing blog

Reply:
This has been requested to remove.
<img width="586" alt="image" src="https://user-images.githubusercontent.com/57550290/195371376-6af6d149-b796-4264-904a-8c11ca831c08.png">